packages/website/docs/parseAlgoliaHitReverseSnippet.md

```diff
@@ -20,20 +48,20 @@ const hit = {
     },
   },
 };
-const snippetParts = parseAlgoliaHitReverseSnippet({
+const reverseSnippetedParts = parseAlgoliaHitReverseSnippet({
   hit,
   attribute: 'name',
 });
 
-// => [{ value: 'Lap', isHighlighted: false }, { value: 'top', isHighlighted: true }]
+// [{ value: 'Lap', isHighlighted: false }, { value: 'top', isHighlighted: true }]
 ```
 
 ## Example with nested attributes
 
 ```js
 import { parseAlgoliaHitReverseSnippet } from '@algolia/autocomplete-preset-algolia';
 
-// Fetch an Algolia hit
+// An Algolia hit for query "lap"
```

Comment:
We can use the same nested example as the other pages.

Reply:
We use the query "lap" for all the `parse*` docs. I agree it's better to go with the new ones. Let's merge this as is and I can do a global PR that fixes them all, this will be easier than switching from branch to branch.